android/src/main/java/com/theoplayer/ReactTHEOplayerContext.kt

```diff
@@ -321,6 +325,23 @@ class ReactTHEOplayerContext private constructor(
     } catch (e: Exception) {
       Log.w(TAG, "Failed to configure Cast integration ${e.message}")
     }
+    try {
+      if (BuildConfig.EXTENSION_MEDIA3) {
+        @Suppress("UnstableApiUsage")
+        media3Integration =
+          Media3PlayerIntegrationFactory.createMedia3PlayerIntegration { _, _ ->
+            // selectedSource -> represents the TypedSource the player picked to play.
+            // source -> represents the SourceDescription passed to the player.
+            // return true -> the Media3 integration pipeline will be used to play the selected source.
+            // return false -> the default pipeline will be used to play the selected source.
+            configAdapter.useMedia3
+          }
+        playerView.player.addIntegration(media3Integration)
+      }
+    } catch (e: Exception) {
+      Log.w(TAG, "Failed to configure Cast integration ${e.message}")
```

Comment:
This should probably log something like `"Failed to configure Media3 integration ${e.message}"` instead?